tests/unittest_nodes.py

```diff
@@ -1583,6 +1583,15 @@ def func():
     """
     )
     assert node.doc == "Docstring"
+    assert isinstance(node.doc_node, nodes.Const)
+    assert node.doc_node.lineno == 3
+    assert node.doc_node.col_offset == 4
+    if PY38_PLUS:
+        assert node.doc_node.end_lineno == 3
+        assert node.doc_node.end_col_offset == 15
+    else:
+        assert node.doc_node.end_lineno is None
+        assert node.doc_node.end_col_offset is None
```

Comment:
Could you add a multiline example? I'm would like to check that `end_lineno` is still correct

Reply:
added